src/types.ts

```diff
@@ -98,7 +98,7 @@ export interface Overrides {
   traits: Trait[]
   perks: Perk[]
   base_stats?: BaseStat[]
-  filter_match?: boolean
+  filter_match?: number // index of rule, or undefined/-1 if didn't match any?
```

Comment:
Comment is a little unclear - it can only ever be -1, or a positive int now right? If it can be undefined and I've not spotted how, we should type this accordingly.

Reply:
I guess this was mostly a mental note for myself while making the feature 😅 Initially I had it as undefined but the current version would only be undefined if rbf is turned off. But when it's enabled there's always a value for this field available.